dist/robots.cfg

```diff
@@ -114,6 +123,8 @@ RobotUA <<EOR
     pavuk,
     Pinterestbot,
     POE-Component,
+    PetalBot,
+    petalbot,
```

Comment:
Match is case-insensitive, so delete one of these.

Reply:
done